.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ commands:
     parameters:
       version:
         type: string
-        default: 1.31.0
+        default: 1.42.0
```

Comment:
Is this needed or just updating while we are here?

Reply:
Just updating while we are here.

Reply:
Alright, as long as it doesn't cause new errors we should be good :)

Reply:
You are right, might need a cache key bump.